governance/polls/WBTC Risk Premium Adjustment - August 24, 2020

```diff
@@ -0,0 +1,42 @@
+---
+title: WBTC Risk Premium Adjustment - August 24, 2020
+summary: Signal your support to adjust the WBTC Risk Prmium within a range of 0% to 6%
+discussion_link: https://forum.makerdao.com/t/2406
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: 0
+   2: 1
+   3: 2
+   4: 3
+   5: 4
+   6: 5
+   7: 6
+---
+# Poll: WBTC Risk Prmium Adjustment - August 24, 2020
+
+The Maker Foundation Interim Risk Team has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) which presents a number of possible **WTBC Risk Premium** options. Voters are now able to signal their support for a **WTBC Risk Premium** within a range of 0% to 6%.
```

Comment:
The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll to express support or opposition...